xrootd/xrdfuse/xrdfuse.go

```diff
@@ -0,0 +1,166 @@
+// Copyright 2018 The go-hep Authors.  All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Package xrdfuse contains implementation of a FUSE api for the XRootD.
```

Comment:
`s/contains implementation of a FUSE api for the XRootD./contains the implementation of the FUSE API accessing a remote filesystem served over the XRootD protocol./`